src/otx/algo/callbacks/adaptive_train_scheduling.py

```diff
@@ -32,6 +33,7 @@ def __init__(self, max_interval: int = 5, decay: float = -0.025):
         self.decay = decay
         self._saved_check_val_every_n_epoch: int | None = None
         self._saved_log_every_n_steps: int | None = None
+        self.is_enabled: bool = False
```

Comment:
Is this needed?

Reply:
Fixed, https://github.com/openvinotoolkit/training_extensions/pull/2857/commits/4a6cd2ba0bb1e526bd1c181b56a1db0c076685ee